core/conversion/converters/impl/detach.cpp

```diff
@@ -0,0 +1,30 @@
+#include "NvInfer.h"
+#include "core/conversion/converters/converters.h"
+#include "core/util/prelude.h"
+#include "torch/torch.h"
+
+namespace trtorch {
+namespace core {
+namespace conversion {
+namespace converters {
+namespace impl {
+namespace {
+
+auto detach_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns().pattern(

```

Comment:
Do we need to do anything for a detach op since there are no gradients? Can we just alias tensor IDs?

Reply:
It might be cleaner to lower these ops away like `aten::to`